src/components/atoms/AreaDetails/AreaDetails.tsx

```diff
@@ -43,8 +50,16 @@ export const AreaDetails: React.FC<AreaDetailsProps> = ({ areaSize, firstAnalysi
                         <td>
                             <p>Amount of timestamps:</p>
                         </td>
-                        <td>{timestepsCount}</td>
+                        <td>{timestampsCount}</td>
                     </tr>
+                    {timestampWithSignificantPlastic && (
+                        <tr>
+                            <td>
+                                <p>Number of days without plastic found:</p>
+                            </td>
+                            <td>{timestampsCount - timestampWithSignificantPlastic}</td>
+                        </tr>
+                    )}
```

Comment:
Are we only showing this if timestampWithSignificantPlastic > 0? Wouldn't it be a useful information to know that there there was no day without any plastic detected?:)

Reply:
we are always showing it, even when there are 0 days without plastic :) 
The conditional rendering is just to make sure that we only display this if we have the information about the `timestampWithSignificantPlastic` which we should have usually :)